slices/liblttng-ust1t64.yaml

```diff
@@ -1,4 +1,7 @@
 package: liblttng-ust1t64
+essential:
+  - liblttng-ust1t64_copyright
+
```

Comment:
nit: it would be nice to have a blank line in between `package` and `essential`.

Reply:
done